src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -58,37 +57,10 @@ const ResultsPane: FC = () => {
   }
 
   const submit = () => {
-    let timeRangeStart, timeRangeStop
-
-    if (!timeRange) {
-      timeRangeStart = timeRangeStop = null
-    } else {
-      if (timeRange.type === 'selectable-duration') {
-        timeRangeStart = '-' + timeRange.duration
-      } else if (timeRange.type === 'duration') {
-        timeRangeStart = '-' + timeRange.lower
-      } else if (isNaN(Date.parse(timeRange.lower))) {
-        timeRangeStart = null
-      } else {
-        timeRangeStart = new Date(timeRange.lower).toISOString()
-      }
-
-      if (!timeRange.upper) {
-        timeRangeStop = 'now()'
-      } else if (isNaN(Date.parse(timeRange.upper))) {
-        timeRangeStop = null
-      } else {
-        timeRangeStop = new Date(timeRange.upper).toISOString()
-      }
-    }
```

Comment:
This code is only used to extract a `timeRangeStart` and `timeRangeStop` from the TimeRange abstraction. So can override an override.
* there is lower down code, which incorrectly overrides the TimeRange abstraction.

Reply:
took out the changes to the ResultsPane and notebooks. Figure we can have that discussion, after this PR goes in (and fixes everywhere else in the UI application).